RussianFilter/sections/css_extended.txt

```diff
@@ -1,6 +1,9 @@
 !
 ! Extended CSS rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/36245
+adultmult.xyz##.title_h2:has( > a[href^="https://bp2.infiplay.com/"])
+adultmult.xyz##div:has( > a[href^="https://www.vsemayki.ru/"])
```

Comment:
Hiding the link is enough, it does not save any space when I remove the div too.

For the other two ext-CSS rules, please use #?#

Reply:
I suppose that `##a[target="_blank"][rel="nofollow"] > img` can be even better in this case, isn't it?